packages/commandkit/package.json

```diff
@@ -47,7 +48,6 @@
     "devDependencies": {
         "@types/node": "^20.5.9",
         "discord.js": "^14.13.0",
-        "dotenv": "^16.3.1",
```

Comment:
yeah we kind of fucked up last time not realizing this lmao

Reply:
Shouldn't have really affected the bottom line though right? It was just there for the tests and it's a dev dep

Reply:
cli needs this package